packages/esm-active-visits-app/src/visits-summary/visit-detail.test.tsx

```diff
@@ -1,21 +1,23 @@
 import React from 'react';
 import userEvent from '@testing-library/user-event';
 import { render, screen } from '@testing-library/react';
-import { formatDate } from '@openmrs/esm-framework';
 import { useVisit } from './visit.resource';
 import VisitDetailComponent from './visit-detail.component';
 
+const mockUseVisit = jest.mocked(useVisit);
 const defaultProps = {
   patientUuid: '691eed12-c0f1-11e2-94be-8c13b969e334',
   visitUuid: '497b8b17-54ec-4726-87ec-3c4da8cdcaeb',
 };
-const mockUseVisit = jest.mocked(useVisit);
 
-jest.mock('./visit.resource');
```

Comment:
Complete module mocking is an anti-pattern that triggers single-spa warnings in the console output. Prefer selective mocking to:

- Mock only what's necessary for your test
- Preserve real behavior for unchanged module parts
- Use actual implementations where possible for better test coverage and reliability